solr/modules/monitor/src/test-files/solr/collection1/solrconfig.xml

```diff
@@ -0,0 +1,173 @@
+  <requestHandler name="/query" class="org.apache.solr.monitor.search.ReverseSearchHandler"/>
+  <requestHandler name="/browse" class="solr.SearchHandler"
+    useParams="query,facets,velocity,browse">
+    <lst name="defaults">
+      <str name="echoParams">explicit</str>
+    </lst>
+  </requestHandler>
+  <requestHandler name="/pingQuery" class="solr.SearchHandler">
+    <lst name="invariants">
+      <str name="q">0</str>
+      <str name="log.debug">false</str>
+      <str name="wt">xml</str>
+      <str name="df">_version_</str>
+    </lst>
+  </requestHandler>
+  <requestHandler name="/admin/ping" class="solr.PingRequestHandler">
+    <lst name="invariants">
+      <str name="q">0</str>
+      <str name="qt">/pingQuery</str>
+    </lst>
+    <lst name="defaults">
+      <str name="echoParams">all</str>
+    </lst>
+  </requestHandler>
+
+  <searchComponent name="reverseSearch"
+    class="org.apache.solr.monitor.search.ReverseSearchComponent">
+    <str name="presearcherType">TermFilteredPresearcher</str>
+  </searchComponent>
+  <searchComponent name="spellcheck" class="solr.SpellCheckComponent">
+    <str name="queryAnalyzerFieldType">text_general</str>
+  </searchComponent>
+  <searchComponent name="tvComponent"
+    class="org.apache.solr.handler.component.TermVectorComponent"/>
+  <requestHandler name="/tvrh" class="solr.SearchHandler" startup="lazy">
+    <lst name="defaults">
+      <bool name="tv">true</bool>
+    </lst>
+    <arr name="last-components">
+      <str>tvComponent</str>
+    </arr>
+  </requestHandler>
+  <searchComponent name="terms" class="solr.TermsComponent"/>
+  <requestHandler name="/terms" class="solr.SearchHandler" startup="lazy">
+    <lst name="defaults">
+      <bool name="terms">true</bool>
+      <bool name="distrib">false</bool>
+    </lst>
+    <arr name="last-components">
+      <str>terms</str>
+      <str>tvComponent</str>
+    </arr>
+  </requestHandler>
+  <searchComponent name="elevator" class="solr.QueryElevationComponent">
+    <str name="queryFieldType">string</str>
+  </searchComponent>
+
+  <requestHandler name="/elevate" class="solr.SearchHandler" startup="lazy">
+    <lst name="defaults">
+      <str name="echoParams">explicit</str>
+    </lst>
+    <arr name="last-components">
+      <str>elevator</str>
+    </arr>
+  </requestHandler>
+  <schemaFactory class="ClassicIndexSchemaFactory"/>
+  <updateRequestProcessorChain>
+    <processor class="solr.LogUpdateProcessorFactory"/>
+    <processor class="solr.DistributedUpdateProcessorFactory"/>
+    <processor class="org.apache.solr.monitor.update.MonitorUpdateProcessorFactory"/>
+    <processor class="solr.RunUpdateProcessorFactory"/>
+  </updateRequestProcessorChain>
+  <queryResponseWriter name="json" class="solr.JSONResponseWriter">
+    <str name="content-type">text/plain; charset=UTF-8</str>
+  </queryResponseWriter>
+  <queryResponseWriter name="velocity" class="solr.VelocityResponseWriter" startup="lazy">
```

Comment:
minor: `VelocityResponseWriter` got removed (or relocated) in https://issues.apache.org/jira/browse/SOLR-14792

subjective: suggest to prune `solrconfig.xml` content to remove anything that's not needed in the context of the monitor module

Reply:
yes, I agree.. I will take a look at that.

Reply:
I've started this process .. will prune some more probably